specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -221,7 +221,7 @@ export function MenuButton({
     p-[1.4vh]
     ${
       isActive
-        ? 'bg-brand-300 text-white'
+        ? 'bg-brand-300 !text-white'
```

Comment:
do you also need to add `!` on line 226 or is that good as is?

Reply:
it's good at it is